components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/SubscriptionsApiServiceImpl.java

```diff
@@ -77,60 +77,6 @@ public Response blockSubscription(String subscriptionId, String blockState, Stri
                 RestApiUtil.handleResourceNotFoundError(RestApiConstants.RESOURCE_SUBSCRIPTION, subscriptionId, log);
             }
 
-            Application subscribedApp = currentSubscription.getApplication();
-            String applicationTokenType = "OAUTH";
-            if (subscribedApp != null) {
-                applicationTokenType = subscribedApp.getTokenType();
-            }
-
-            //in case of a JWT type application add a subscription blocking condition as well.
-            if (APIConstants.APPLICATION_TOKEN_TYPE_JWT.equals(applicationTokenType)) {
-                Identifier apiId = currentSubscription.getApiId();
-                if (apiId == null) {
-                    apiId = currentSubscription.getProductId();
```

Comment:
is this change compatible with master branch?

Reply:
This change was introduced to master branch in this PR https://github.com/wso2/carbon-apimgt/pull/10937/files#diff-522c8c043470c5cf678afa86a710f36bc44b95afe8f4052142ff5af1ca9a70a4